relevanceai/recipes/__init__.py

```diff
@@ -0,0 +1,2 @@
+from relevanceai.recipes.base import Workflow
```

Comment:
rename `recipes` folder to `workflow` (remember - non plural)

`sentimentops` -> goes inside `operation`

Reply:
Renamed recipe to workflow